modules-magic-drc/README.src.rst

```diff
@@ -0,0 +1,24 @@
+``skywater-pdk-actions`` - ``modules-magic-drc``
```

Comment:
Is anything here specific to skywater-pdk modules?

I'm thinking that this action would be better called something like `check-gds-for-sky130-drc-with-magic`, `check-layouts-for-sky130-drc-with-magic`?